turtlebot3c-bringup-snap/snap/local/core_launcher.sh

```diff
@@ -7,4 +7,13 @@ if [[ -z "${TURTLEBOT3_MODEL}" ]]; then
     export TURTLEBOT3_MODEL="$(snapctl get turtlebot3-model)"
 fi
 
-ros2 launch turtlebot3_bringup robot.launch.py
+SIMULATION="$(snapctl get simulation)"
+
+case ${SIMULATION} in
+  false|0)
+    ros2 launch turtlebot3_bringup robot.launch.py
+    ;;
+  true|1)
+    ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py simulation:="${SIMULATION}"
```

Comment:
I think the parameter is `use_sim_time` and not `simulation`: https://github.com/ROBOTIS-GIT/turtlebot3/blob/humble-devel/turtlebot3_bringup/launch/turtlebot3_state_publisher.launch.py#L31

Reply:
You're right, this was obviously a test :P

Reply:
Fixed.